ui/src/track-layout/layout-switch-api.ts

```diff
@@ -98,19 +105,20 @@ export async function getSwitches(
 }
 
 export async function getSwitchJointConnections(
-    publishType: PublishType,
+    layoutContext: LayoutContext,
     id: LayoutSwitchId,
 ): Promise<LayoutSwitchJointConnection[]> {
     return getNonNull<LayoutSwitchJointConnection[]>(
-        `${layoutUri('switches', publishType, id)}/joint-connections`,
+        `${layoutUri('switches', layoutContext, id)}/joint-connections`,
     );
 }
 
 export async function insertSwitch(
     newSwitch: TrackLayoutSwitchSaveRequest,
+    layoutContext: LayoutContext,
 ): Promise<Result<LayoutSwitchId, TrackLayoutSaveError>> {
     const apiResult = await postNonNullAdt<TrackLayoutSwitchSaveRequest, LayoutSwitchId>(
-        layoutUri('switches', 'DRAFT'),
+        layoutUri('switches', draftLayoutContext(layoutContext)),
```

Comment:
Tämä on tavallaan ihan ymmärrettävä: tietenkin kaikki muutokset aina draftiin. Toisaalta jos tänne joku sanoo että insert official contextilla, olisi ehkä parempi saada virhe, koska jossain muualla lienee jotain jännästi. Backari ei taida tarjota post ../official/.. polkuja kuitenkaan joten vain passaamalla context sellaisenaan, tämä varmaankin antaisi virheen.

Reply:
Joo, nämä on nykyisellään kyllä vähän outoja. En tosiaan koske näihin tämän enempää tässä yhteydessä, mutta olen kyllä kanssa nykyään sillä kannalla että näille voisi vaan passata suoraan `publicationState`:n sellaisenaan - tulisiphana samalla nähtyä jos jossain koitetaankin muokata juttuja `OFFICIAL`:in kautta, koska nyt siitä ei tosiaan virhettä tule